rpcs3/Emu/Cell/lv2/sys_timer.cpp

```diff
@@ -294,14 +294,28 @@ error_code sys_timer_usleep(ppu_thread& ppu, u64 sleep_time)
 
 	sys_timer.trace("sys_timer_usleep(sleep_time=0x%llx)", sleep_time);
 
-	u64 passed = 0;
+	if (sleep_time)
+	{
+		u64 passed = 0;
+		u64 remaining;
+
+		lv2_obj::sleep(ppu, sleep_time);
+
+		while (sleep_time > passed)
+		{
+			remaining = sleep_time - passed;
 
-	lv2_obj::sleep(ppu, std::max<u64>(1, sleep_time));
+			if (remaining > 500)
```

Comment:
Add a comment about why we use 500 here.  Or use a const with a meaningful variable name.